airdrop/processor/base_airdrop.py

```diff
@@ -11,3 +11,10 @@ def match_signature(address, formatted_message, signature):
         if recovered_address.lower() == address.lower():
             signature_verified = True
         return signature_verified, recovered_address
+
+    @staticmethod
+    def trim_prefix_from_string_message(prefix, message):
```

Comment:
We don’t have a generic utility already a available  be for this ?

Reply:
Didn't had time to clean up the util. That's why I kept it separate.
Will raise separate PR on cleaning Util